.github/workflows/ci.yml

```
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10.0-alpha.7"]
+        python-version: ["3.8", "3.9", "3.10", "3.11-dev"]
```

`3.11` final and `3.12-dev` are now available:

```suggestion
        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12-dev"]
```

Hopefully the dependencies are available for 3.12, let's see what the CI says.